crates/compiler/can/src/desugar.rs

```diff
@@ -479,21 +535,28 @@ pub fn desugar_expr<'a>(
         Closure(loc_patterns, loc_ret) => arena.alloc(Loc {
             region: loc_expr.region,
             value: Closure(
-                desugar_loc_patterns(arena, loc_patterns, src, line_info, module_path),
-                desugar_expr(arena, loc_ret, src, line_info, module_path),
+                desugar_loc_patterns(arena, loc_patterns, src, line_info, module_path, problems),
+                desugar_expr(arena, loc_ret, src, line_info, module_path, problems),
             ),
         }),
         Backpassing(loc_patterns, loc_body, loc_ret) => {
             // loc_patterns <- loc_body
             //
             // loc_ret
 
+            let problem_region = Region::span_across(
```

Comment:
Currently we only highlight the `foo, bar <- try parseResult` part of the backpassing, I think it's too much to include the body of the function but I'm open to discussion on this.